doc/classes/HBoxContainer.xml

```diff
@@ -16,6 +17,9 @@
 	</constants>
 	<theme_items>
 		<theme_item name="separation" type="int">
+			Seperation is a parameter unique to HBoxContainer and VBoxContainer.
+			Enabling this will allow the user to specify the seperation distance in pixels between the contained elements.
+			You can find this setting in the inspector under the Custom Constants rollout in the Control heirlooms.
```

Comment:
We don't usually specify how to find properties in Inspector in the documentation. Some generic doc page about how to go from a class reference "theme item" to the Inspector could be useful, but I wouldn't start putting it in all classref pages.

"Control heirlooms" is also a very unusual way to refer to Control's inherited properties.